tests/src/test/scala/com/nvidia/spark/rapids/timezone/TimeZoneSuite.scala

```diff
@@ -160,18 +180,24 @@ class TimeZoneSuite extends SparkQueryCompareTestSuite {
           .set("spark.sql.datetime.java8API.enabled", "true"))
 
     // get result from TimeZoneDB
-    val actualRet = withResource(createColumnVector(epochSeconds)) { inputCv =>
-      TimeZoneDB.fromUtcTimestampToTimestamp(
-        inputCv,
-        ZoneId.of(zoneStr))
+    withResource(createColumnVector(epochSeconds)) { inputCv =>
+      val actualRet = if (useGPU) {
+        GpuTimeZoneDB.fromUtcTimestampToTimestamp(
```

Comment:
Can we merge ```GpuTimezoneDB``` together ```TimezoneDB``` together? And we can switch runtime via a configuration.